arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/GeoAggregateRESTService.java

```diff
@@ -395,24 +396,30 @@ public Response geoaggregatePost(
     private FeatureCollection getFeatureCollection(MixedRequest request, CollectionReference collectionReference, boolean flat) throws ArlasException, IOException {
         FeatureCollection fc;
         AggregationResponse aggregationResponse = new AggregationResponse();
+        AggregationTypeEnum maintAggregationType = ((AggregationsRequest)request.basicRequest).aggregations.get(0).type;
```

Comment:
Why do you mean with `maint`?

Reply:
It's a typo, I meant main aggregation , the first bucket aggregation